src/main/resources/swagger/api-docs.yaml

```diff
@@ -3633,14 +3665,66 @@ components:
           example: my-managed-resource-group
       description: specifies a request for a pet managed identity
     User:
+      type: object
       required:
+        - id
         - email
-      type: object
+        - enabled
+        - createdAt
+        - updatedAt
       properties:
+        id:
+          type: string
+          description: User's Id
+        googleSubjectId:
+          type: string
+          description: User's Google subject Id
         email:
           type: string
           description: User's email address
+          format: email
+        azureB2CId:
+          type: string
+          description: User's Azure B2C Id
+        enabled:
+          type: boolean
+          description: Whether or not the user is enabled
+        acceptedTosVersion:
+          type: string
+          description: The version of terms of service the user accepted
+        createdAt:
+          type: string
+          format: date-time
+          description: User's time of creation
+        registeredAt:
+          type: string
+          format: date-time
+          description: User's time of registration
+        updatedAt:
+          type: string
+          format: date-time
+          description: User's time of last update
       description: specification of a User
+    UpdateUserRequest:
+      type: object
+      properties:
+        googleSubjectId:
+          type: string
+          description: User's Google subject Id
+        email:
+          type: string
+          description: User's email address
+          format: email
+        azureB2CId:
```

Comment:
Are we sure we want the B2C ID to be updatable?

Reply:
As it stands right now, the only field that is updatable in practice is the email. The UpdateUserRequest object does contain all of these fields and I realized that the swagger doc was not in sync with the UpdateUser endpoint's request object. We still need to make decisions on what fields we are allowing to update. I think ultimately that the B2CId might be useful to update without having to hit the db directly. We can restrict updating this field to only superadmins or something. Right now, we are using the sam user directives to manage authorization and the level needed to use this endpoint are admins. Going forward, I think we will have a permissions service that had handle what actions are allowed by a user more granularly (nobody can update a user email except for themselves and users with admin roles)